api/http/api_deployments.go

```diff
@@ -976,7 +987,7 @@ ParseLoop:
 			}
 			// Add one since this will impose the upper limit on the
 			// artifact size.
-			if size > d.config.MaxImageSize {
```

Comment:
just to double check, for what are we now using `MaxImageSize`?
also I do not understand the comment with "Add one"

Reply:
> also I do not understand the comment with "Add one"

I have no idea. It seems like this may have been a comment for a different if-branch from the past? :man_shrugging: